### ISSUE TITLE
Provide both raw and view URLs for github studies / collections

### ISSUE THREAD

**Issue by kcranston** [OpenTreeOfLife/peyotl#149]
See [opentree issue #871](https://github.com/OpenTreeOfLife/opentree/issues/871). We've implemented a fix on the UI side for collections, but if we were going to provide similar 'view study in github' URLs (i.e. that point to the pretty github version rather than the raw version), would it make sense to have peyotl provide both a raw URL and view URL rather than hack the `external_url` property for the UI?

**Comment by jimallman** [comment]
Agreed. I'm also pondering good general names to distinguish these:
 - `history_url` or `github_url` or `repo_url` for the full GitHub view?
 - `download_url` or `view_url` or `raw_url` for the "raw" latest version?

**Comment by jimallman** [comment]
OK to close this? Or should we press on with uniform standards for "raw" and GitHub views of all resource types in peyotl?
